### DISCUSSION TITLE
Hello I have trouble understanding why 3 tests are not passing.

### DISCUSSION THREAD

**Discussion by YavorJJ** [Cyfrin/foundry-full-course-cu#1800] [Q&A] [answered]
' ' ' solidity 
// SPDX-License-Identifier: MIT 

pragma solidity ^0.8.19;

import {Test, console} from "forge-std/Test.sol";
import {FundMe} from "../src/FundMe.sol";
import {DeployFundMe} from "../script/DeployFundMe.s.sol";

contract FundMeTest is Test {
  FundMe fundMe;

  function setUp() external {
    //fundMe = new FundMe(0x694AA1769357215DE4FAC081bf1f309aDC325306);
    DeployFundMe deployFundMe = new DeployFundMe();
    fundMe = deployFundMe.run();
  }

  function testMinimumDollarIsFive() public {
    assertEq(fundMe.MINIMUM_USD(), 5e18);
  } 

  function TestOwnerIsMsgSender () public  {
    assertEq(fundMe.i_owner(), msg.sender);
  }

  function testPriceFeedVersionIsAccurate() public {
     uint256 version = fundMe.getVersion();
     assertEq(version, 4);
  }
}

Ran 2 tests for test/Testt.sol:FundMeTest
[PASS] testMinimumDollarIsFive() (gas: 8323)
[PASS] testPriceFeedVersionIsAccurate() (gas: 19079)
Suite result: ok. 2 passed; 0 failed; 0 skipped; finished in 2.62s (877.09ms CPU time)

why the third wont pass which is: TestOwnerIsMsgSender

**Comment by heheboii11** [comment, marked as answer]
Use smaller case t in the function, it should be like 
```
function testOwnerIsMsgSender () public {assertEq(fundMe.i_owner(), msg.sender);
}
```

**Reply by YavorJJ** [reply]
3 of them passed 🫡, much respect bro. Thank you!

**Comment by EngrPips** [comment]
Hello @YavorJJ, Please try @heheboii11  suggestion, and let's see if it resolves your issue.

**Reply by YavorJJ** [reply]
yes, it did! thank you :)

**Reply by EngrPips** [reply]
Awesome, We have a very helpful community! Impressive
